crates/node/tests/identity.rs

```diff
@@ -0,0 +1,97 @@
+use calimero_node::{Node, NodeConfig};
+use calimero_primitives::context::ContextId;
+use camino::Utf8PathBuf;
+use libp2p::identity::Keypair;
+use tokio::sync::{broadcast, oneshot};
+use {calimero_context, calimero_network, calimero_server, calimero_store};
+
+fn create_test_config() -> NodeConfig {
+    let identity = Keypair::generate_ed25519();
+    NodeConfig {
+        home: Utf8PathBuf::from("/tmp"),
```

Comment:
this would fail on macOS and Windows

Reply:
I'll update tests once I know the source code is all correct 🙂